src/JsonFieldData/Readme.md

```diff
@@ -16,7 +16,7 @@ When you install the JSON plugin on your AQTS app server, it is recommended that
 
 | AQTS Version | Latest compatible plugin Version |
 | --- | --- |
-| AQTS 2025.2 | [v25.2.2](https://github.com/AquaticInformatics/aquarius-field-data-framework/releases/download/v25.2.2/JsonFieldData.plugin) |
+| AQTS 2025.2 | [v25.2.3](https://github.com/AquaticInformatics/aquarius-field-data-framework/releases/download/v25.2.2/JsonFieldData.plugin) |
```

Comment:
Build doesn't exist yet, but this will need to be updated to the future published build link
```suggestion
| AQTS 2025.2 | [v25.2.3](https://github.com/AquaticInformatics/aquarius-field-data-framework/releases/download/v25.2.3/JsonFieldData.plugin) |
```